web-common/src/features/templates/types.ts

```diff
@@ -50,10 +50,26 @@ export interface MarkdownTemplateT {
   markdown: MarkdownProperties;
 }
 
+export interface ImageAdjust {
+  opacity?: number;
+  blur?: number;
+  saturation?: number;
+}
+export interface ImageProperties {
+  url: string;
+  size?: string;
```

Comment:
Instead of this being a generic `string` type. Better to mention the strings that can be used explicitly here.
` fill | contain | cover | none | scale-down `

Reply:
The above typing will also help with autocomplete and type hints when we add that.